### PR TITLE
Add fb tags

### PR DESCRIPTION
**Pull request by maretamara**
Dodanie tagów facebookowych do podstron z ogłoszeniami o pracę
* [ ] @korzonek 
* [ ] @pjar 
* [x] @beatadelura 
* [ ] @MagdaMalinowska 
* [ ] @AdamMankowski 
* [ ] @rpietraszko 
* [ ] @kruszczynski

### CODE REVIEW

File: lib/custom_helpers.rb [hunk 1]
```diff
@@ -18,8 +18,11 @@ def blog?
     current_page.path == 'blog.html'
   end
 
-  def elixir?
-    current_page.path == 'join-our-crew/elixir.html'
+  def render_fb_tags_partial
+    page_name = current_page.path.split('/').last.split('.').first
+    partial "partials/fb_tags_#{page_name}"
+  rescue
```
Comment (kruszczynski) [inline lib/custom_helpers.rb]: It's good practice to provide the error you want to rescue from so that you're not accidentally rescuing real errors.

Comment (MagdaMalinowska) [inline lib/custom_helpers.rb]: Agree but middleman errors are weird:P We can only check it by message :/

Comment (kruszczynski) [inline lib/custom_helpers.rb]: Yes, but in this case you can rescue all Middleman partial errors.

Comment (kruszczynski) [inline lib/custom_helpers.rb]: Also, maybe we can use a `Hash` to fetch template name then we can yield a block with default value.

Comment (MagdaMalinowska) [inline lib/custom_helpers.rb]: @kruszczynski https://github.com/codequest-eu/website/pull/238/commits/0656d53d8f9f69edc12683e8b6a744be88fe634d
I used array to define existing fb_tags partials. Is it ok or you were thinking about sth better?:)

**Pull request closed without merging**